src/main/java/org/jitsi/jicofo/rest/Health.java

```diff
@@ -253,10 +253,7 @@ private static int cacheStatus(int status)
         JitsiMeetServices services
             = focusManager.getJitsiMeetServices();
 
-        Assert.notNull(services, "services");
```

Comment:
The reason I've put those here was to get a message in NPE indicating which variable is null(to know even without looking at the line number and checking the code). I don't insist that those are necessary just explaining my reasoning behind that. If you think it's not of much use I'm good with having those deleted.